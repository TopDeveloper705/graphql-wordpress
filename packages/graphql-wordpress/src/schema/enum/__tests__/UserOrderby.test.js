import USER_ORDERBY from 'enum/UserOrderby';

describe('Test enum values', () => {
  test('USER_ORDERBY values', () => {
    expect(USER_ORDERBY.getValues()).toMatchSnapshot();
  });
});
