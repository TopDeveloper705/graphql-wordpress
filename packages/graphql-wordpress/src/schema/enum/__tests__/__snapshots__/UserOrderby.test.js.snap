// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test enum values USER_ORDERBY values 1`] = `
Array [
  Object {
    "deprecationReason": undefined,
    "description": undefined,
    "isDeprecated": false,
    "name": "ID",
    "value": "id",
  },
  Object {
    "deprecationReason": undefined,
    "description": undefined,
    "isDeprecated": false,
    "name": "INCLUDE",
    "value": "include",
  },
  Object {
    "deprecationReason": undefined,
    "description": undefined,
    "isDeprecated": false,
    "name": "NAME",
    "value": "name",
  },
  Object {
    "deprecationReason": undefined,
    "description": undefined,
    "isDeprecated": false,
    "name": "REGISTERED_DATE",
    "value": "registered_date",
  },
  Object {
    "deprecationReason": undefined,
    "description": undefined,
    "isDeprecated": false,
    "name": "SLUG",
    "value": "slug",
  },
  Object {
    "deprecationReason": undefined,
    "description": undefined,
    "isDeprecated": false,
    "name": "EMAIL",
    "value": "email",
  },
  Object {
    "deprecationReason": undefined,
    "description": undefined,
    "isDeprecated": false,
    "name": "URL",
    "value": "url",
  },
]
`;
